src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/WorkloadManifest.json.in

```diff
@@ -8,6 +8,7 @@
       "description": ".NET WebAssembly build tools",
       "packs": [
         "Microsoft.NET.Runtime.WebAssembly.Sdk",
+        "Microsoft.NET.Sdk.WebAssembly.Pack",
```

Comment:
should we merge this into Microsoft.NET.Runtime.WebAssembly.Sdk? or rename because it's no longer coming from the SDK repo?

Reply:
I don't have strong opinion about the name. I struggled with this name last year. The package is essential part of WasmSDK, but content lives in runtime repo 🤷‍♂️

About merging, longer term Runtime WasmSDK will be merged into WasmSDK pack, or the Runtime could be renamed to NativeBuild/Relink or something, as that will be all that will remain there. But the cleanup hasn't been done yet. The wasmconsole isn't still using WasmSDK